modules/misc/gtk.nix

```
@@ -166,14 +166,26 @@ in {
     home.packages = optionalPackage cfg.font ++ optionalPackage cfg.theme
       ++ optionalPackage cfg.iconTheme;
 
-    home.file.${cfg2.configLocation}.text =
-      concatStringsSep "\n" (mapAttrsToList formatGtk2Option ini) + "\n"
-      + cfg2.extraConfig;
+    home.file.${cfg2.configLocation} = {
+      text = concatStringsSep "\n" (mapAttrsToList formatGtk2Option ini) + "\n"
+        + cfg2.extraConfig;
+      onChange = mkIf (cfg.theme != null)
+        (let xsettingsd = "${pkgs.xsettingsd}/bin/xsettingsd";
+        in ''
+          timeout 2 ${xsettingsd} -c <(echo 'Net/ThemeName "${cfg.theme.name}"') & true
```

```suggestion
          timeout 2 ${xsettingsd} -c <(echo 'Net/ThemeName "${cfg.theme.name}"') &
```
`true` is useless here

---

I would suggest
```suggestion
          timeout 2 ${xsettingsd} -c <(echo 'Net/ThemeName "${cfg.theme.name}"') &> /dev/null &
```
to avoid messy and confusing output during activation.

---

Thanks!